frontend/src/components/ConfirmationDialog.scss

```diff
@@ -0,0 +1,94 @@
+/* ConfirmationDialog.css */
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1000;
+}
+
+.modal-content {
+  background-color: white !important;
+  border-radius: 8px;
+  width: 100%;
+  max-width: 60vw;
+  padding: 24px;
```

Comment:
We'll need to use the values from the Design Tokens list as much as possible in our styles:
https://www2.gov.bc.ca/gov/content/digital/design-system/foundations/design-tokens/glossary
For example, this line would be `padding: var(--layout-padding-large);`
(turning the dots into dashes: "layout.padding.large" -> "--layout-padding-large")

Check out the existing .scss files in the frontend for some examples. Let me know if you want a hand and we can divide and conquer these files tomorrow